dts/bindings/can/atmel,sam-can.yaml

```diff
@@ -9,6 +9,14 @@ include:
 properties:
   reg:
     required: true
+    description: |
+      2 reg blocks needed; Register block for the MCAN configuration registers;
+      Register block for the DMA Base Address, pointing to Message RAM Area
```

Comment:
I would leave out the ", pointing to Message RAM Area" part, as it can be confusing whether this is the address of the message RAM.